files/prometheus_wrapper

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
I don't think the wrapper is needed since we are using Gosu, should probably remove all references to it.